app/__init__.py

```diff
@@ -1,22 +1,26 @@
 from flask import Flask
 from .db import db, migrate
 from .models import task, goal
+from .routes.task_routes import bp as tasks_bp
+from .routes.goal_routes import bp as goals_bp
 import os
 
-def create_app(config=None):
+def create_app(config=None): # Factory function, creates and returns Flask app
```

Comment:
```suggestion
def create_app(config=None):
```

Comments can be helpful for understanding a new project, however, when you're in industry a pull request with explanatory comments left in like this might not be reviewed until you've removed them.

Every comment added needs to be maintained so you're increasing the lines of code that need to be updated. These comments are unnecessary for the codebase since folks with experience working on backend APIs can review documentation to understand what's going on.

In the future, I'd recommend removing comments before you open a pull request. If you really want to keep your comments, then create a new branch and add the comments there. I'd love to see folks wean off of comments as we move through the program so you can get a better feel for reviewing documentation to understand code. This will also provide practice with writing code that will be committed to a professional, production codebase.

Reply:
Thank you for the comment. Since this is a learning project, I initially added those comments as personal notes to help me remember the purpose of each part of the code. I will follow your advice going forward.

Now: Resolved.